src/app/components/Headings/index.jsx

```diff
@@ -46,3 +49,15 @@ export const SubHeading = styled.h2`
     line-height: 2.25em;
   }
 `;
+
+export const SubHeading = ({ text }) => (
```

Comment:
We don't need to do this and can just set the `tabIndex` and `id` as `attrs` of the [SubHeading styled component](https://github.com/bbc/simorgh/pull/458/files#diff-47d43d5e0c8d431009435db20cd0d1abR34). See https://www.styled-components.com/docs/api#attrs